game/scripts/npc/items/item_yasha_and_kaya_5.txt

```diff
@@ -71,23 +71,23 @@
       }
       "03"
       {
-        "var_type"              "FIELD_INTEGER"
+        "var_type"                                        "FIELD_INTEGER"
         "bonus_attack_speed"                              "16"
       }
       "04"
       {
-        "var_type"              "FIELD_INTEGER"
-        "manacost_reduction"                              "12 18 26 34 42" // OAA
+        "var_type"                                        "FIELD_INTEGER"
+        "manacost_reduction"                              "16 18 26 34 42"
       }
       "05"
       {
-        "var_type"                      "FIELD_INTEGER"
-        "movement_speed_percent_bonus"                    "8 12 16 20 24" // OAA
```

Comment:
this was intentional, revert this. Movement speed is tier locked because otherwise you can replace boots super trivially.